qa/rpc-tests/feature_fedpeg.py

```diff
@@ -229,11 +264,24 @@ def run_test(self):
         print("Test pegout")
         self.test_pegout(get_new_unconfidential_address(parent), sidechain)
 
-        print("Test pegout P2SH")
-        parent_chain_addr = get_new_unconfidential_address(parent)
-        parent_pubkey = parent.validateaddress(parent_chain_addr)["pubkey"]
-        parent_chain_p2sh_addr = parent.createmultisig(1, [parent_pubkey])["address"]
-        self.test_pegout(parent_chain_p2sh_addr, sidechain)
+        # print("Test pegout P2SH")
```

Comment:
why?

Reply:
read the commits, perhaps better read the commit tittles in the branch without the squashes: https://github.com/jtimon/elements/tree/pr-415

Reply:
please don't disable swaths of tests when it's very possible to retain them